workshops/personal_website/README.md

```diff
@@ -11,87 +11,79 @@ workshop will take around 45 minutes.
-Think Dropbox, but for code. We'll be using it to store and manage our code in
-Hack Club.
+We'll be using it to store and manage our code in Hack Club.
 
 1. Open https://github.com/join in a new tab and create a new account
-  - Choose a username you will be proud to show future employers (ex.
-    `zachlatta` and `jonleung`)
+  - Choose a username you won't regret. Your actual name is a solid bet.
   - Use an email that you have access to, you'll need to access it
     to verify your account
-  - Make sure to remember your username and password, you'll need this at every
+  - Make sure to remember your username and password, you'll need them at every
     club meeting
   - When prompted to choose a plan, choose the **"Free"** plan
-2. Open your email inbox in a new tab and look for an email from GitHub. Open it
-   and click the button to verify your email
+2. Verify your email by checking the inbox of the email you used
 3. Open https://gh.hackclub.com in a new tab and star the project by clicking
    the button that looks like this on the top right: ![](img/github_star.png)
 
 ### 2) Create Your First GitHub Repository
 
-GitHub allows us to **host** our website using a service called **GitHub
-Pages**. This means that we can put the files of our website on GitHub and
-GitHub will give us a URL that we can share with the world.
+GitHub allows us to host our website using a service called GitHub Pages. This
+means that we can put the files of our website on GitHub and GitHub will give
+us a URL that we can share with the world.  
 
 1. Go to https://github.com
-2. Click the green **"+ New repository"** button
-3. Under **"Repository name"** write your `USERNAME.github.io` except instead of
-   writing `USERNAME`, write your actual GitHub username. So if your username is
-   `alice1337`, write `alice1337.github.io`. You **have** to name your
-   repository this in order for GitHub Pages to work correctly (we'll talk more
-   about GitHub Pages later)
+2. Click the green "+ New repository" button
+3. Under "Repository name" write `USERNAME.github.io` except instead of writing
+   `USERNAME`, write your actual GitHub username. So if your username is
+   `alice1337`, write `alice1337.github.io`. You have to name your repository
+   this in order for GitHub Pages to work correctly (we'll talk more about
+   GitHub Pages later)
 4. Make sure that you did the previous step correctly.
-  - Is your **username** entered correctly
-  - The repository name should end in **`.io`**
+  - Is your username entered correctly
+  - The repository name should end in `.io`
 5. After you are sure you entered in the repository name correctly, click the
    green "Create repository" button at the bottom.
-6. Copy the **"HTTPS link"** at the top. It should look like this:
-   `https://github.com/username/username.github.io.git`. You will need this link
-   for the next step.
+6. You've created your first repository! Yay. Now, copy the "HTTPS link"
```

Comment:
I always feel like words like "yay" ending with a period feels... insincere. :wink: I'd word it as: "Yay! You've created your very first repository."